crates/agglayer-jsonrpc-api/src/tests/send_certificate.rs

```diff
@@ -327,3 +327,62 @@ async fn pending_certificate_in_error_force_set_status() {
     assert!(res.settlement_tx_hash.is_some());
     assert_eq!(res.status, CertificateStatus::Candidate);
 }
+
+#[test_log::test(tokio::test)]
+async fn pending_certificate_in_error_with_settlement_tx_hash_force_set_status() {
```

Comment:
Also a test for attempting remove tx hash from settled certificate would be nice

Reply:
> Do you mean, to confirm it fails as expected?

Yep.